src/System.Diagnostics.Process/tests/ProcessTests.netcoreapp.cs

```diff
@@ -40,5 +40,30 @@ public void Start_StandardInputEncodingPropagatesToStreamWriter()
 
             Assert.Same(encoding, process.StandardInput.Encoding);
         }
+
+        [Fact]
+        public void StartProcessWithArgumentList()
+        {
+            List<string> argList = new List<string>();
+            argList.Add("arg1");
+            argList.Add("arg2");
+
+            ProcessStartInfo psi = new ProcessStartInfo(GetCurrentProcessName(), argList);
+            Process testProcess = CreateProcess();
+            testProcess.StartInfo = psi;
+
+            try
+            {
+                testProcess.Start();
+                Assert.Equal("arg1 arg2", testProcess.StartInfo.Arguments);
```

Comment:
It'd be good for the test to validate the arguments were actually passed to the target process as expected.